clouddriver-core/src/main/groovy/com/netflix/spinnaker/clouddriver/model/NoopRawResourceProvider.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2020 Coveo, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.clouddriver.model;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Set;
+
+public class NoopRawResourceProvider implements RawResourceProvider {
```

Comment:
Do we need this `NoopRawResourceProvider` as it looks like the `RawResourceController` handles the absence of a `RawResourceProvider` just fine.  (I think these exist for other providers in cases where the auto-wiring isn't robust to a missing provider.)

Reply:
Removed